docs/Concepts/Permissioning/Plugin-Permissioning.md

```diff
@@ -0,0 +1,20 @@
+---
+description: Plugin based permissioning
+---
+
+# Plugin based permissioning
+
+Plugin based permissioning allows complex permissioning solutions to be defined by extending besu.
+To do this you need to [build a plugin](../Concepts/Plugins.md).
+
+This is currently broken down into two parts; connection permissioning and message permissioning.
```

Comment:
Hi @antonydenyer, regarding configuring this plugin. I assume the plugin must implement the [`PermissioningService` service](https://github.com/hyperledger/besu/blob/9c1d44076a969c8a6b2ab9cf17198b8ccca7359d/plugin-api/src/main/java/org/hyperledger/besu/plugin/services/PermissioningService.java)?

Reply:
Yes, that's correct. I guess the question is how much technical detail should we put in here vs high level overview?

Reply:
It's tricky because we don't have an obvious place to put this content yet. But if there's no example code to reference, then we should probably provide enough information to get started.